test/WebSites/RazorWebSite/Services/TestBodyTagHelperComponent.cs

```diff
@@ -15,7 +15,7 @@ public override Task ProcessAsync(TagHelperContext context, TagHelperOutput outp
         {
             if (string.Equals(context.TagName, "body", StringComparison.Ordinal) && output.Attributes.ContainsName("inject"))
             {
-                output.PostContent.AppendHtml("<script>'This was injected!!'</script>");
+                output.PostContent.AppendHtml("<script>'This was injected!!'</script>\r\n");
```

Comment:
Do not add Windows-specific line endings. My preference is to leave this unchanged because this changes the baselines for no reason.